types/index.ts

```diff
@@ -44,13 +25,7 @@ export interface IPrecompiled {
 }
 
 export interface IOpcodeMeta {
-  input: string
-  output: string
-  description: string
-}
-
-export interface IPrecompiledMeta {
-  name: string
+  name?: string
```

Comment:
I think it would help a lot if we added comments on when we expect this to be filled or not